changelogs/unreleased/fix-misc-memory-leaks.md

```diff
@@ -0,0 +1,14 @@
+## bugfix/box
+
+- Fixed memory leak on disconnectiong from replica.
+- Fixed memory leak on user ddl on access check failure.
+- Fixed memory leak on xlog open failure.
+- Fixed memory leak on space ddl on certain condition.
+- Fixed memory leak on iproto override usage (since 3.1).
+- Fixed memory leak on ddl failed due to foreign key constraint check.
+- Fixed memory leak on space drop with sequence bound to json path.
```

Comment:
```suggestion
- Fixed a memory leak on disconnection from a replica.
- Fixed a memory leak on user DDL on access check failure.
- Fixed a memory leak on `xlog` open failure.
- Fixed a memory leak on space ddl on certain condition.
- Fixed a memory leak on iproto override usage (since 3.1).
- Fixed a memory leak on DDL failed due to foreign key constraint check.
- Fixed a memory leak on space drop with sequence bound to json path.
```